Resources/Prototypes/Entities/Mobs/Player/dragon.yml

```diff
@@ -0,0 +1,121 @@
+- type: entity
+  name: space dragon
+  id: MobDragon
+  description: A flying leviathan, loosely related to space carps. Go ahead, ask it why it can talk.
+  components:
+  - type: Damageable
+    damageContainer: Biological
+  - type: GhostTakeoverAvailable
+    makeSentient: true
+    name: Space dragon!
+    description: Crash, roast, flood the station with carps!
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Physics
+    bodyType: KinematicController
+  - type: HeatResistance
+  - type: CombatMode
+    disarmAction:
+      name: Disarm
+      enabled: false
+      autoPopulate: false
+  - type: Internals
```

Comment:
does it really need that

Reply:
Otherwise it gets the disarm action

Reply:
Adding a dummy action isn't a good solution, make it null or something.